docs/windows/deployment/overview.md

```diff
@@ -0,0 +1,191 @@
+
+## Publish
+
+At this time, publishing is only supported through `msbuild`, not `dotnet`.
+
+To publish your app, open the **Developer Command Prompt for VS 2022 Preview** terminal and navigate to the project's folder. Run `msbuild` in publish mode:  
+
+```console
+msbuild /restore /t:Publish /p:TargetFramework=net6.0-windows10.0.19041 /p:configuration=release
+```
+
+The following table defines the parameters used by the previous command:
+
+| Parameter                  | Value                                                                             |
+|----------------------------|-----------------------------------------------------------------------------------|
+| `/restore`                 | Restores any dependencies referenced by the project.                              |
+| `/t:Publish`               | Runs the publish command.                                                         |
+| `/p:TargetFramework`       | The target framework, which is a Windows TFM, such as `net6.0-windows10.0.19041`. |
+| `/p:configuration=Release` | Sets the build configuration, which is `Release`.                                 |
+
+Publishing builds and packages the app, copying the signed package to the _bin\\Release\\net6.0-windows10.0.19041\\win10-x64\\AppPackages\\\<appname>\\_ folder. Where \<appname> is a folder named after both your project and version. In this folder there's an _msix_ file, that's the app package.
+
+## Installing the app
+
+To install the app, it must be signed with a certificate that you already trust. If it isn't, Windows won't let you install the app. You'll be presented with a dialog similar to the following, with the Install button disabled:
+
+:::image type="content" source="media/overview/install-untrusted.png" alt-text="Installing an untrusted app.":::
+
+Notice that in the previous image, the Publisher was "unknown."
+
+To trust the certificate of app package, perform the following steps:
+
+01. Right-click on the _.msix_ file and choose **Properties**.
+01. Select the **Digital Signatures** tab.
+01. Choose the certificate then press **Details**.
+
+    :::image type="content" source="media/overview/properties-digital-signatures.png" alt-text="Properties pane of an MSIX file with the digital signatures tab selected.":::
+
+01. Select **View Certificate**.
+01. Select **Install Certificate...**
+01. Choose **Local Machine** then select **Next**.
+
+    If you're prompted by User Account Control to **Do you want to allow this app to make changes to your device?**, select **Yes**.
+
+01. In the **Certificate Import Wizard** window, select **Place all certificates in the following store**.
+01. Select **Browse...** and then choose the **Trusted People** store. Select **OK** to close the dialog.
+
+    :::image type="content" source="media/overview/certificate-import.png" alt-text="Certificate import wizard window is shown while selecting the Trusted People store.":::
```

Comment:
Are you sure this screenshot is correct? I mean the actual contents on the screenshot here. I think it has to be imported into the "Trusted Root Certification Authorities" store.  But seeing how you got the Install button to light up below, I guess I learned something today ;)